### PR TITLE
Handle auth token of default test user implicitly

### PR DESCRIPTION
**Pull request by dmytro-ndp**
This PR implements handling of auth token of default user implicitly in Codenvy on-prem selenium tests (issue https://github.com/codenvy/qa/issues/1106)

@tolusha: please, review this request.

### CODE REVIEW

File: selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClient.java [hunk 1]
```diff
@@ -38,21 +39,36 @@
 
     private final String                 apiEndpoint;
     private final HttpJsonRequestFactory requestFactory;
+    private final TestUser adminTestUser;
 
     @Inject
     public OnpremTestOrganizationServiceClient(TestApiEndpointUrlProvider apiEndpointUrlProvider,
-                                               HttpJsonRequestFactory requestFactory) {
+                                               HttpJsonRequestFactory requestFactory,
```
Comment (tolusha) [inline selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClient.java]: It think we need to have separate Service Client for admin operations

Comment (dmytro-ndp) [inline selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClient.java]: Agree.

File: selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClientForAdmin.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import org.eclipse.che.selenium.core.provider.TestApiEndpointUrlProvider;
+import org.eclipse.che.selenium.core.requestfactory.TestHttpJsonRequestFactoryForAdmin;
+
+/**
+ * This util is handling the requests to Organization API.
+ */
+@Singleton
```
Comment (tolusha) [inline selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClientForAdmin.java]: If we have several implementations it is recommended to use `Named` annotation for binding.

File: selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClientForAdmin.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+public class OnpremTestOrganizationServiceClientForAdmin extends OnpremTestOrganizationServiceClient {
```
Comment (tolusha) [inline selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClientForAdmin.java]: I prefer these names:
```
     OnpremTestAdminOrganizationServiceClient
     OnpremTestUserOrganizationServiceClient
```

Comment (dmytro-ndp) [inline selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClientForAdmin.java]: Those are better names.

File: selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClientForAdmin.java (hunk 2 above)
Comment (dmytro-ndp) [inline selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClientForAdmin.java]: Agree.

File: selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/OnpremSeleniumSuiteModule.java [hunk 4]
```diff
@@ -104,4 +110,11 @@ public TestWorkspace getWorkspace(TestWorkspaceProvider testWorkspaceProvider,
     public ActionsFactory getActionFactory() {
         return isMac() ? new MacOSActionsFactory() : new GenericActionsFactory();
     }
+
+    @Provides
+    @Named("admin")
+    public OnpremTestOrganizationServiceClient getAdminOrganizationServiceClient(TestApiEndpointUrlProvider apiEndpointUrlProvider,
+                                                                                 TestAdminHttpJsonRequestFactory requestFactory) {
```
Comment (tolusha) [inline selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/OnpremSeleniumSuiteModule.java]: Where `TestAdminHttpJsonRequestFactory` is bind?

Comment (dmytro-ndp) [inline selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/OnpremSeleniumSuiteModule.java]: We doesn't need to bind it explicitly so as we have concrete class **TestAdminHttpJsonRequestFactory** here and _Guice_ knows where it should inject it from.

**Pull request merged**